src/MMM01.md

```diff
@@ -0,0 +1,190 @@
+
+MMM01 starts up in a unique mode ("unmapped") which always maps the _last 32 kiB_ of the ROM to the 0000-7FFF address region regardless of the values in the bank/mode registers. The correct ROM header (with Nintendo logo) therefore needs to be located at offset `(size - 32 kiB) + 100h` in the ROM rather than the usual `0000 + 100h` (which contains the header of the first game in the collection instead). MMM01 cartridges have a simple menu program in this last 32 kiB, which manipulates the additional MMM01 control bits, allowing game selection and setting the game size, before entering "mapped" mode and booting the selected game (see "Mapping Enable" below).
+
+As the last 32 kiB of the ROM are reserved for the cartridge menu, it's best to pack games into the ROM from largest to smallest to avoid having a game overlap the menu. e.g. the Taito Variety Pack contains three 128 kiB games and one 64 kiB game, leaving 32 kiB unused and 32 kiB for the menu in a 512 kiB ROM chip.
+
+## Memory
+
+### 0000-3FFF - ROM Bank X0 (Read Only)
+
+On start (in "unmapped" mode), this is mapped to the first half of the menu program in the last 32 kiB of the ROM.
+
+When a game is mapped, this area normally contains the first 16 KiB (bank 00) of the game ROM. In multiplexed mode with MBC1 mode set to 1, banks 20h, 40h, and 60h of the game ROM can be mapped here (the same as MBC1).
+
+### 4000-7FFF - ROM Bank 01-7F (Read Only)
+
+On start (in "unmapped" mode), this is mapped to the second half of the menu program in the last 32 kiB of the ROM.
+
+When a game is mapped, this area may contain any of the further 16 kiB banks of the ROM. Cannot address any banks where the masked main ROM banking register would be 00, instead it automatically maps to the bank 1 higher.
+
+### A000-BFFF - RAM Bank 00-03, if any (Read/Write)
+
+This area is used to address external SRAM in the cartridge (if any). The RAM is only accessible if RAM is enabled, otherwise reads return open bus values (often $FF, but not guaranteed) and writes are ignored.
+
+External RAM is often battery-backed, allowing for the storage of game data while the Game Boy is turned off, or if the cartridge is removed from the Game Boy.
+
+It is currently unknown whether RAM access is possible while in unmapped mode.
+
+## Registers
+
+The MMM01 registers are extensions of the MBC1 registers. In "mapped" mode, the extra bits are unwriteable and the MMM01 emulates an MBC1. In "unmapped" mode, the extra bits are writeable, allowing the menu program to select which game to play and configure its size and RAM access.
+
+All the MMM01 registers are 7-bits in size and set to 00h on power-up. For the ROM Bank Number register, this behaves as if it was set to 01h.
+
+### 0000-1FFF - RAM Enable (Write Only)
+
+```
+Bits: X 6 5 4 3 2 1 0
```

Comment:
Can you show me an example of what you mean?

Reply:
[This](https://gbdev.io/pandocs/STAT.html#ff41---stat-lcd-status-rw).

Reply:
That's... really ugly.

Reply:
[We know.](https://github.com/gbdev/pandocs/issues/89)

Reply:
Can we not keep the diagrams? The table loses the "visual" aspect of it.

I copied the concept from the tauwasser's MMM01 docs and found it very readable there.

Reply:
If you think the format should be changed (and yes, monospaced text must definitely be removed for this purpose), please open an issue so that we can decide on what replacement is best.

In the meantime, consistency is more important, if only so as to make conversion later on less painful.

Reply:
I made one: #318 (I can't seem to label it as "consistency/style")

Reply:
That's because only maintainers can modify labels, of course ;)

Reply:
While we wait for #318 to be resolved (which will entail making some custom markup, so it'll take time), can those be changed to the same template as the rest of the document? They'll be converted in due time, and it's not worth letting this PR rot just for this.

Reply:
I am currently away